store/src/java/com/zimbra/cs/dav/resource/ScheduleOutbox.java

```diff
@@ -128,6 +130,9 @@ public void handlePost(DavContext ctxt) throws DavException, IOException, Servic
                 String val = prop.getValue();
                 if (val != null) {
                     organizer = val.trim();
+                    if (StringUtil.isNullOrEmpty(delegationInfo.getOriginator())) {
```

Comment:
trying to understand how is this schedule code responsible for handling freebusy check